app/webpacker/styles/registration.scss

```diff
@@ -0,0 +1,7 @@
+.confirmed-banner {
+  background-color: #00703c;
+  padding: 50px 30px;
+  margin: 15px;
+  text-align: center;
+  color: govuk-colour("white") !important;
+}
```

Comment:
I imagine you took the styles from a site like https://design-system.service.gov.uk/patterns/confirmation-pages/

I guess that means we don't have them in our govuk styling by default?

Reply:
No, I just made the styles and wasn't aware of  https://design-system.service.gov.uk/patterns/confirmation-pages/. I'll try and use the classes from there as we probably already have the styles.